### DISCUSSION TITLE
Keep the form visible after submission

### DISCUSSION THREAD

**Discussion by mikewink** [tobimori/kirby-dreamform#66] [Feature Requests]
Hello TobiMori,

I wanted to ask if there is a setting to keep the form visible after submission and just empty the fields? Have I missed it somewhere?

In our current design, the form is long and when it is submitted, and removed, it creates a big layout shift. 

Thank you.

**Comment by tobimori** [comment]
Thanks for the suggestion. I moved this from issues to the feature requests section, since this is not possible right now.

You could try to solve it by implementing a custom form.php snippet and remove this line: https://github.com/tobimori/kirby-dreamform/blob/develop/snippets/form.php#L91
